src/addmult.c

```diff
@@ -42,6 +42,7 @@ GPtrArray *mults_possible;
 enum { ALL_BAND, PER_BAND };
 
 char mult1_value[40];
+int mult1_mode = PER_BAND;  //FIXME configure
```

Comment:
Is this a hard coded variable? I agree that we don't need to handle this one through config, but may be that wouldn't be a big work. Would I add it?

Reply:
Not a big work indeed, but for the time being I'd not commit to a config name as the general concept hasn't been worked out yet. (e.g. there shall be at least 2 mults, hence the name `mult1_...`)
IMO  for the moment it's enough to mark it as fixme.